docs/getting_started/tutorial.md

```diff
@@ -146,6 +146,19 @@ After the server starts, go to <http://127.0.0.1:8000> to see Wagtail’s welcom
 ```{note}
 This tutorial uses `http://127.0.0.1:8000` as the URL for your development server but depending on your setup, this could be a different IP address or port. Please read the console output of `manage.py runserver` to determine the correct URL for your local site.
 ```
+### Running and Writing Tests
+
+Wagtail now includes basic tests in the `home/tests.py` file when you create a new project using `wagtail start`. These tests check:
+
+- If the **Root Page** (ID=1) is automatically created.
+- If the **Homepage** is created as a child of the root page.
+- If a **BlogIndexPage** (or other page types) can be added.
```

Comment:
Just a refinement, avoiding bold and using the model as a code formatted proper noun.

```suggestion
-   If the _root `Page`_ (ID=1) is automatically created.
-   If the _home `Page`_ is created as a child of the root page.
-   If a `BlogIndexPage` (or other page types) can be added.
```

Also, adding extra spaces to the list, while we do not use Prettier in MD docs, it's going to get these extra spaces when we do manual clean up sometimes.